README.md

```diff
@@ -1,7 +1,7 @@
 boolean\_expression: a small Rust crate for Boolean expressions and BDDs
 ========================================================================
 
-[![Build Status](https://travis-ci.org/cfallin/boolean_expression.svg?branch=master)](https://travis-ci.org/cfallin/boolean_expression)
+![Rust](https://github.com/domoritz/boolean_expression/workflows/Rust/badge.svg)
```

Comment:
This seems to be pointing to your fork -- s/domoritz/cfallin/ ?

Reply:
Ups, copy pasta error.